terraform/aws/variables.tf

```diff
@@ -109,3 +109,8 @@ variable "amis" {
     us-west-2 = "ami-00657261"
   }
 }
+
+variable "docker_version" {
+  description = "Docker version"
+  default = "1.8.2-0~trusty"
```

Comment:
ideally should merge this one for the sake of consistency then tag so we got an snapshot with new weave approach plus the same version of docker across the cluster, then create a new pr for new version